package.json

```diff
@@ -8,18 +8,13 @@
     "build": "tsc && rsbuild build",
     "start": "rsbuild dev",
     "test": "vitest",
-    "format": "biome format --write src",
-    "lint": "biome lint ./src",
-    "check": "biome check --apply src",
-    "check-unsafe": "biome check --apply-unsafe src",
+    "lint": "biome check ./src",
+    "format": "biome check --apply src",
+    "format-unsafe": "biome check --apply-unsafe src",
     "preview": "rsbuild preview"
   },
   "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
+    "production": [">0.2%", "not dead", "not op_mini all"],
```

Comment:
Why is this getting formatted from inline to multi-line and vice versa every couple of PRs?
Are we somehow using different format rules?